recipes/physx/4.1.1/conanfile.py

```diff
@@ -0,0 +1,266 @@
+        self._cmake.definitions["PXSHARED_PATH"] = os.path.abspath(os.path.join(self._source_subfolder, "pxshared")).replace("\\", "/")
+        self._cmake.definitions["PXSHARED_INSTALL_PREFIX"] = self.package_folder.replace("\\", "/")
+        self._cmake.definitions["PX_GENERATE_SOURCE_DISTRO"] = False
+
+        # Options defined in externals/cmakemodules/NVidiaBuildOptions.cmake
+        self._cmake.definitions["NV_APPEND_CONFIG_NAME"] = False
+        self._cmake.definitions["NV_USE_GAMEWORKS_OUTPUT_DIRS"] = False
+        if self.settings.compiler == "Visual Studio":
+            self._cmake.definitions["NV_USE_STATIC_WINCRT"] = str(self.settings.compiler.runtime).startswith("MT")
+            self._cmake.definitions["NV_USE_DEBUG_WINCRT"] = str(self.settings.compiler.runtime).endswith("d")
+        self._cmake.definitions["NV_FORCE_64BIT_SUFFIX"] = False
+        self._cmake.definitions["NV_FORCE_32BIT_SUFFIX"] = False
+        self._cmake.definitions["PX_ROOT_LIB_DIR"] = os.path.abspath(os.path.join(self.package_folder, "lib")).replace("\\", "/")
+
+        if self.settings.os == "Windows":
+            # Options defined in physx/source/compiler/cmake/windows/CMakeLists.txt
+            self._cmake.definitions["PX_COPY_EXTERNAL_DLL"] = False # External dll copy disabled, PhysXDevice dll copy is handled afterward during conan packaging
+            self._cmake.definitions["PX_FLOAT_POINT_PRECISE_MATH"] = self.options.enable_float_point_precise_math
+            self._cmake.definitions["PX_USE_NVTX"] = False # Could be controlled by an option if NVTX had a recipe, disabled for the moment
+            self._cmake.definitions["GPU_DLL_COPIED"] = True # PhysXGpu dll copy disabled, this copy is handled afterward during conan packaging
+
+            # Options used in physx/source/compiler/cmake/windows/PhysX.cmake
+            self._cmake.definitions["PX_GENERATE_GPU_PROJECTS"] = False
+
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _get_physx_build_type(self):
+        if self.settings.build_type == "Debug":
+            return "debug"
+        elif self.settings.build_type == "RelWithDebInfo":
+            return "checked"
+        elif self.settings.build_type == "Release":
+            if self.options.release_build_type == "profile":
+                return "profile"
+            else:
+                return "release"
+
+    def _get_target_build_platform(self):
+        return {
+            "Windows" : "windows",
+            "Linux" : "linux",
+            "Macos" : "mac",
+            "Android" : "android",
+            "iOS" : "ios"
+        }.get(str(self.settings.os))
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        tools.save(os.path.join(self.package_folder, "licenses", "LICENSE"), self._get_license())
+
+        out_lib_dir = os.path.join(self.package_folder, "lib", self._get_physx_build_type())
```

Comment:
is this really needed? there is `cmake.install` above...

Reply:
It is required because PhysX's cmake.install puts libs in lib/<build_type>, not in lib or bin.

Reply:
A `tools.move` utility function would be useful.